src/assets/styles/theme.styles.ts

```diff
@@ -20,5 +20,13 @@ export const theme: Theme = {
       grey: '#ddd',
       lightgrey: '#aaa',
     },
+    background: {
+      grey: '#edeff1',
+      white: '#fff',
+    },
+  },
+  width: {
+    small: '991px',
+    mobile: '786px',
```

Comment:
Generalnie spoko, tylko czy nazwy `small` i `mobile` do siebie pasują? To albo piszemy `small`, `medium`, `large`, albo `mobile`, `tablet` i `desktop` (ta pierwsza konwencja jest bardziej popularna)